docs/pyenv-scl.md

```diff
@@ -0,0 +1,121 @@
+# Kapitan on Older Operating Systems
+
+## Introduction
+
+Kapitan requires Python 3.6, and you need to be able to install the dependencies in the 
+[requirements file](https://github.com/deepmind/kapitan/blob/master/requirements.txt) file.  However, sometimes this isn't entirely straightforward and you may not be able or willing to install new versions of Python system-wide.  
```

Comment:
A relative link here would withstand time better than an absolute one. :) https://blog.github.com/2013-01-31-relative-links-in-markup-files/